client/app/components/form/titles/operationsTitles.tsx

```diff
@@ -0,0 +1,29 @@
+export const StatutoryDeclarationDisclaimerTitle = (
+  <>
+    <b>Please note:</b>
+    <br />
+    <ul className="mt-0">
+      <li>
+        Information in this application will be shared with the Minister of
+        Finance and the name of the Operator and BORO ID may be disclosed.
+      </li>
+      <li>
+        The Operator is responsible for determining their eligibility for an
+        exemption under the Carbon Tax Act.
+      </li>
+    </ul>
+  </>
+);
+
+export const StatutoryDeclarationUploadFieldTitle = (
+  // Uncomment and replace the URL with the correct one when the template is ready in #645
+  // <>
+  //   Statutory Declaration
+  //   <br />
+  //   <span className="font-normal">
+  //     (Download <a href="TEMPLATE DOWNLOAD URL HERE">template</a>)
+  //   </span>
+  // </>
+  // Delete this line when the template is ready
+  <>Statutory Declaration</>
```

Comment:
Can we comment this back in and just have the link lead nowhere for now? It would be good for people to able to see this is done